runtime/onert/backend/gpu_cl/open_cl/MemoryManagement.cc

```diff
@@ -111,7 +111,6 @@ absl::Status AssignObjectsToTensors(const std::vector<TensorUsageRecord<uint3>>
     default:
       return absl::InternalError("MemoryStrategy is not supported with current tensor size type.");
   }
-  return absl::OkStatus();
```

Comment:
Above functions have same issue.

Reply:
Sorry i don't understand...
Can you explain a little more to me?

Reply:
ah.... line 99 and line 84?... oh Thanks!